src/olympia/amo/management/commands/check_locales_completion_rate.py

```diff
@@ -0,0 +1,175 @@
+    # and we want to keep at least one of those to ensure AMO works in RTL.
+    LOCALES_TO_ALWAYS_KEEP = (
+        'ar',
+        'de',
+        'en-US',
+        'es-ES',
+        'fr',
+        'he',
+        'it',
+        'ja',
+        'pl',
+    )
+    EMAIL_RECIPIENTS = ('amo-locales-notifications@mozilla.com',)
+    EMAIL_SUBJECT = 'AMO locales completion rate check'
+    EMAIL_TEMPLATE_PATH = 'amo/emails/locales_completion.ltxt'
+
+    def add_arguments(self, parser):
+        """Handle command arguments."""
+        parser.add_argument(
+            '--dry-run',
+            action='store_true',
+            dest='dry_run',
+            default=False,
+            help='Fetch data from Pontoon but do not send emails.',
+        )
+
+    def handle(self, *args, **options):
+        locales_below_threshold, locales_above_threshold = (
+            self.find_locales_below_and_above_threshold()
+        )
+        self.warn_about(
+            locales_below_threshold=locales_below_threshold,
+            locales_above_threshold=locales_above_threshold,
+        )
+
+    def find_locales_below_and_above_threshold(self):
+        locales_below_threshold = set()
+        locales_above_threshold = set()
+        seen_locales = defaultdict(int)
+        for project in self.PONTOON_PROJECTS:
+            data = self.fetch_data(project)
+            for locale_data in data.get('project', {}).get('localizations', []):
+                locale = self.get_locale(locale_data)
+                seen_locales[locale] += 1
+                completion = self.get_completion(locale_data)
+                # If we see a locale below treshold, immediately add it to the
+                # relevant set.
+                if completion < self.COMPLETION_THRESHOLD:
+                    self.stdout.write(f'❌ {locale} is below threshold ({completion}%)')
+                    locales_below_threshold.add(locale)
+                # If we see a language above threshold but not already enabled
+                # in production, add it to the relevant set.
+                elif locale not in settings.AMO_LANGUAGES:
+                    self.stdout.write(f'✅ {locale} is above threshold ({completion}%)')
+                    locales_above_threshold.add(locale)
+        # Add to locales below threshold locales we only saw in one project.
+        # Use the ones we already have enabled in production, in case somehow
+        # a locale would disappear completely from pontoon and still be enabled
+        # on our side.
+        for locale in settings.AMO_LANGUAGES:
+            if locale != settings.LANGUAGE_CODE and seen_locales[locale] != len(
+                self.PONTOON_PROJECTS
+            ):
+                self.stdout.write(
+                    f'❌ {locale} is only in {seen_locales[locale]} project(s) !'
+                )
+                locales_below_threshold.add(locale)
+        # Somewhat inefficient but good enough check to remove any locales that
```

Comment:
Nit: You have sets so you could use set intersection to get a more efficient filter?

```python
duplicate_locales = locales_above_threshold & locales_below_threshold
locales_above_threshold = locales_above_threshold - duplicate_locales
```